### ISSUE TITLE
Problem when requesting a compatibility OpenGL context on iOS

### ISSUE THREAD

**Issue by tanis2000** [rust-windowing/glutin#1145]
I'm currently asking Glutin for a Compatibility profile on iOS (just like I do on macOS where it works like a charm):

```
        let mut el = glutin::EventsLoop::new();
        let wb = glutin::WindowBuilder::new()
        .with_title("minigame-rust")
        .with_dimensions(glutin::dpi::LogicalSize::new(800.0, 600.0));

        let windowed_context = glutin::ContextBuilder::new()
            .with_vsync(true)
            .with_gl_profile(glutin::GlProfile::Compatibility)
            .build_windowed(wb, &el)
            .unwrap();
```

When I try to compile a shader that is marked as `#version 120` I get a panic:

```
OpenGL version OpenGL ES 3.0 APPLE-17.0.34
GLSL version OpenGL ES GLSL ES 3.00
thread '<unnamed>' panicked at 'ERROR: 0:1: '' :  version '120' is not supported
', src/shader.rs:113:17
```

This happened to me on macOS when I wasn't requesting a Compatibility profile. I then forced it and it worked. I don't know why it's not working on iOS.
Any hints?
Thanks!

**Comment by tanis2000** [comment]
It happens that if I remove the `#version 120` line everything works.

**Comment by ZeGentzy** [comment]
Not familiar with iOS, probably will have to look into it yourself.

**Comment by tanis2000** [comment]
As far as I can tell, if I set no `#version` directive if defaults to OpenGL ES 2.0 / GLSL 1.00 on iOS and Android so it actually works fine.

In theory OpenGL ES 3.0 should be fully compatible with previous versions so unless I set `#version 300` it should work with `#version 120` just like older versions did. I have no clue myself why it's failing, but IMHO that's an OpenGL issue and not directly related to Glutin.
